workbench/src/main/setupAddRemovePlugin.js

```diff
@@ -38,32 +38,34 @@ export function setupAddPlugin() {
         // Create a conda env containing the plugin and its dependencies
         const envName = `invest_plugin_${pluginID}`;
         const mamba = settingsStore.get('mamba');
+      	logger.info(`Plugins: mamba cmd: ${mamba}`);
         let depString = '';
         if (pyprojectTOML.tool.natcap.invest.conda_dependencies) {
           depString = pyprojectTOML.tool.natcap.invest.conda_dependencies.map(
             (dependency) => `"${dependency}"`
           ).join(' ');
         }
-        execSync(
+
+        const createInfo = execSync(
```

Comment:
I was just reading the `execSync` docs and the `maxBuffer` might be an important one to consider. I'm not sure how much stdout there is from these `mamba` commands, but,

>maxBuffer [<number>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Number_type) Largest amount of data in bytes allowed on stdout or stderr. If exceeded, the child process is terminated and any output is truncated. Default: 1024 * 1024.

https://nodejs.org/api/child_process.html#child_processexecsynccommand-options